d3a_interface/constants_limits.py

```diff
@@ -112,6 +112,7 @@ class LoadSettings:
         INITIAL_BUYING_RATE_LIMIT = RangeLimit(0, 10000)
         # Max load energy rate, in ct/kWh
         FINAL_BUYING_RATE_LIMIT = RangeLimit(0, 10000)
+        LOAD_PENALTY_RATE = 29.2
```

Comment:
Please remove these from the d3a.constants. Thanks!

Reply:
Replaced d3a's LOAD_PENALTY_RATE with d3a-interface's LOAD_PENALTY_RATE  but forgot to remove it from d3a.constants, will do while finalising related PR.